src/KafkaFlow.Serializer.NewtonsoftJson/KafkaFlow.Serializer.NewtonsoftJson.csproj

```diff
@@ -9,6 +9,7 @@
 
     <ItemGroup>
       <ProjectReference Include="..\KafkaFlow.Abstractions\KafkaFlow.Abstractions.csproj" />
+      <ProjectReference Include="..\KafkaFlow\KafkaFlow.csproj" />
```

Comment:
I think this dependency is no more needed

Reply:
Removed